testdata/p4_16_samples_outputs/invalid-hdr-warnings2-first.p4

```diff
@@ -86,3 +86,4 @@ control ComputeChecksumI(inout H hdr, inout M meta) {
 }
 
 V1Switch<H, M>(ParserI(), VerifyChecksumI(), IngressI(), EgressI(), ComputeChecksumI(), DeparserI()) main;
+
```

Comment:
Please remove these unrelated changes

Reply:
I will address this while raising review in open source